community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/StatisticsBackedCardinalityModel.scala

```diff
@@ -43,6 +44,10 @@ class StatisticsBackedCardinalityModel(queryGraphCardinalityModel: QueryGraphCar
     case RegularQueryProjection(_, QueryShuffle(_, None, Some(limit: IntegerLiteral))) =>
       Cardinality.min(in, limit.value.toDouble)
 
+    // Normal projection with LIMIT
+    case RegularQueryProjection(_, QueryShuffle(_, None, Some(unknownLimit))) =>
+      Cardinality.min(in, GraphStatistics.DEFAULT_LIMIT_CARDINALITY)
```

Comment:
Why do we guess here instead of avoiding to parametrized constants in limit that we need for planning?